shell/packages/accounts-email-token/token_common.js

```diff
@@ -44,3 +44,21 @@ Router.route("/_emailToken/:_email/:_token", function () {
     }
   });
 });
+
+Router.route("/_emailLinkIdentity/:_email/:_token/:_accountId", function () {
+  this.render("Loading");
+  var self = this;
+  if (Meteor.userId() === this.params._accountId) {
+    Meteor.call("linkEmailIdentityToAccount",
+                this.params._email, this.params._token, this.params._accountId,
```

Comment:
Good catch. Fixed now. Apparently Meteor didn't mind the extra parameter and was still calling the callback upon completion.